packages/devtools_extensions/devtools_extensions.iml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Does this file need to be included?

Reply:
we include the .iml file for the other packages `devtools_app`, `devtools_shared`, `devtools_test`, so following that pattern and including this one as well.